kubernetes/qmgmt-pod.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    role: qmgmt-pod
+    version: 1.3.4
+  name: qmgmt-pod
+spec:
+  containers:
+    - name: qmgmt-pod
+      image: quay.io/quobyte/quobyte-server:1.3.4
+      command:
+        - /bin/bash
+        - -xec
+        - |
+          while true; do sleep 90000; done
```

Comment:
I thought of sleep to minimize the cpu usage but `tail -f /dev/null` seems to work too :)